docs/content/preview/launch-and-manage/monitor-and-alert/xcluster-monitor.md

```diff
@@ -0,0 +1,205 @@
+      role="tab" aria-controls="yugabyted-safe-time" aria-selected="true">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Yugabyted
+    </a>
+  </li>
+  <li>
+    <a href="#local-safe-time" class="nav-link" id="local-safe-time-tab" data-bs-toggle="tab"
+      role="tab" aria-controls="local-safe-time" aria-selected="false">
+      <i class="icon-shell"></i>
+      Local
+    </a>
+  </li>
+</ul>
+<div class="tab-content">
+  <div id="yugabyted-safe-time" class="tab-pane fade show active" role="tabpanel" aria-labelledby="yugabyted-safe-time-tab">
+
+
+```sh
+./bin/yugabyted xcluster status \
+    [--replication_id <replication_id>]
+```
+
+Example output:
+```output
+No Outbound xCluster replications found for this cluster.
+Inbound xCluster Replications:
+
++------------------------------------------------------------------------------------------+
+|                                        yugabyted                                         |
++------------------------------------------------------------------------------------------+
+| Replication 1               :                                                            |
+| Replication ID              : rg1                                                        |
+| State                       : ACTIVE                                                     |
+| Source cluster nodes        : 127.0.0.1,127.0.0.2,127.0.0.3                              |
+| Databases                   : Following are the databases included in this replication   |
+|                             : Database 1:                                                |
+|                             : Name: yugabyte                                             |
+|                             : Safe Time: 2025-03-14 20:43:49.723305                      |
+|                             : Safe Time Lag(micro secs): 0.88                            |
+|                             : Safe Time Skew(micro secs): 0.81                           |
++------------------------------------------------------------------------------------------+
+```
+  </div>
+
+  <div id="local-safe-time" class="tab-pane fade " role="tabpanel" aria-labelledby="local-safe-time-tab">
+
+  ```sh
+  yb-admin \
+      -master_addresses <standby_master_addresses> \
+      get_xcluster_safe_time \
+      [include_lag_and_skew]
+  ```
+
+  Example output:
+  ```output
+  [
+      {
+          "namespace_id": "000034cb000030008000000000000000",
+          "namespace_name": "yugabyte",
+          "safe_time": "2025-03-14 20:51:25.915918",
+          "safe_time_epoch": "1742010685915918",
+          "safe_time_lag_sec": "0.50",
+          "safe_time_skew_sec": "0.02"
+      }
+  ]
+  ```
+
+  </div>
+  </div>
+
+## YB-Master and YB-Tserver UI
+
+You can access the YB-Master and YB-Tserver UIs to monitor the health of xCluster replication at `/xcluster`.
```

Comment:
Maybe link to master and tserver reference pages UI sections

Reply:
Where is this page?